code/modules/hydroponics/plants_crop.dm

```diff
@@ -52,6 +52,7 @@ ABSTRACT_TYPE(/datum/plant/crop)
 	endurance = 0
 	genome = 10
 	commuts = list(/datum/plant_gene_strain/growth_fast,/datum/plant_gene_strain/health_poor)
+	mutations = list(/datum/plantmutation/oat/saltoat)
```

Comment:
```suggestion
	mutations = list(/datum/plantmutation/oat/salt)
```

Reply:
sorry if im spamming your mailbox, idk how it works when I comment on them.